orttraining/orttraining/training_api/include/optimizer.h

```diff
@@ -88,43 +90,9 @@ struct Optimizer {
   std::vector<std::string> input_names_;
   std::vector<std::string> output_names_;
   std::vector<OrtValue> inputs_;
-};
-
-class LearningRateScheduler {
- public:
-  LearningRateScheduler(const Optimizer& optim)
-      : optim_(optim) {
-    ORT_NOT_IMPLEMENTED("Not implemented.");
-  }
-
-  virtual ~LearningRateScheduler() = default;
-
-  // Modify the current learning rate based on current step
-  virtual Status Step(/*int64_t step*/) = 0;
 
-  const Optimizer& optim_;
-};
-
-class LinearScheduler : public LearningRateScheduler {
- public:
-  explicit LinearScheduler(const Optimizer& optim, float start_factor, float end_factor, int64_t total_iters)
-      : LearningRateScheduler(optim),
-        start_factor_(start_factor),
-        end_factor_(end_factor),
-        total_iters_(total_iters) {
-    ORT_NOT_IMPLEMENTED("Not implemented.");
-  }
-
-  // Fetch the step, calculate next value and set lr in optimizer
-  Status Step(/*int64_t step*/) override {
-    ORT_NOT_IMPLEMENTED("Not implemented.");
-    return Status::OK();
-  }
-
- private:
-  float start_factor_;
-  float end_factor_;
-  int64_t total_iters_;
+  // Currently all parameters are in a single group, so we hardcode group0 here.
+  const std::string group_zero_name_ = "group0";
```

Comment:
can this string literal be moved to an unnamed namespace in `optimizer.cc` file?

Reply:
fixed.